internal/pkg/otel/samples/darwin/managed_otlp/platformlogs.yml

```diff
@@ -0,0 +1,67 @@
+receivers:
+  # Receiver for platform specific log files
+  filelog/platformlogs:
+    include: [ /var/log/*.log ]
+    retry_on_failure:
+      enabled: true
+    start_at: end
+    storage: file_storage
+#   start_at: beginning
+
+extensions:
+  file_storage:
+    directory: ${env:STORAGE_DIR}
+
+processors:
+  resourcedetection:
+    detectors: ["system"]
+    system:
+      hostname_sources: ["os"]
+      resource_attributes:
+        host.name:
+          enabled: true
+        host.id:
+          enabled: false
+        host.arch:
+          enabled: true
+        host.ip:
+          enabled: true
+        host.mac:
+          enabled: true
+        host.cpu.vendor.id:
+          enabled: true
+        host.cpu.family:
+          enabled: true
+        host.cpu.model.id:
+          enabled: true
+        host.cpu.model.name:
+          enabled: true
+        host.cpu.stepping:
+          enabled: true
+        host.cpu.cache.l2.size:
+          enabled: true
+        os.description:
+          enabled: true
+        os.type:
+          enabled: true
+
+exporters:
+  # Exporter to print the first 5 logs/metrics and then every 1000th
+  debug:
+    verbosity: detailed
+    sampling_initial: 5
+    sampling_thereafter: 1000
+
+# Exporter to send logs and metrics to Elasticsearch Motel
```

Comment:
Just the naming big with what we discussed internally - `Managed OTLP Input` - [Slack](https://elastic.slack.com/archives/C076MUD9BK8/p1739868979732749?thread_ts=1739806008.640929&cid=C076MUD9BK8)

Applicable in other configs as well.

Reply:
good catch, fixed https://github.com/elastic/elastic-agent/pull/6630/commits/71db336fdac59899b32cbf4dc6e3fad3a760c5fd